server/hwapi/data_models/setup.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024 Canonical Ltd.
+# All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Written by:
+#        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
+
+import os
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from .models import Base
+
+DB_URL = os.getenv("DB_URL", "sqlite://")
+engine = create_engine(DB_URL)
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+DB_PATH = DB_URL.replace("sqlite:///", "")
+if DB_PATH and not os.path.exists(DB_PATH):
+    # Create tables with no data if DB path doesn't exist
+    Base.metadata.create_all(engine)
```

Comment:
I guess this works right now in place of db migrations. But definitely wouldn't work if any schema was changed. So perhaps I'd replace it with migrations

Reply:
I guess this is no longer relevant given your [answer above](https://github.com/canonical/hardware-api/pull/84#discussion_r1549391031)